src/test/java/seedu/module/logic/parser/DeleteTagCommandParserTest.java

```diff
@@ -0,0 +1,4 @@
+package seedu.module.logic.parser;
+
+public class DeleteTagCommandParserTest {
```

Comment:
Here the test case need to be implemented.

Reply:
will implement after we all agree on this new feature in meeting :)

Reply:
was gonna say the same but yes lets discuss